playwright/e2e/settings/encryption-user-tab/encryption-tab.spec.ts

```diff
@@ -93,4 +93,22 @@ test.describe("Encryption tab", () => {
             await checkDeviceIsConnectedKeyBackup(app, expectedBackupVersion, true);
         },
     );
+
+    test("should display the reset identity panel when the user clicks on 'Forgot recovery key?'", async ({
+        page,
+        app,
+        util,
+    }) => {
+        await verifySession(app, "new passphrase");
```

Comment:
FWIW I fixed the bug where the actual recovery key didn't work here, so you can use that if you wish rather than hardcoding the passphrase that the bot client uses.

Reply:
Oh, thanks that's good news. I'll use it then